api.go

```diff
@@ -82,6 +82,12 @@ func CreatePod(podConfig PodConfig) (VCPod, error) {
 		return nil, err
 	}
 
+	// Start the proxy
+	err = p.createProxyProcess()
```

Comment:
You should move that into `p.startVM(netNsPath)` function, so that we don't expand too much `CreatePod()` and `RunPod()`.

Reply:
()Funnily enough, it was there originally, but I moved it out for clarity since `startVM` wasn't just starting the VM so the name was confusing. Also, it matched more closely how we handle the shims (`p.startShims()`).

On the topic of `CreatePod()` and `RunPod()`, I've just raised #486 to remove the duplication so how about we land that first and I can then rebase this PR and there won't be any duplication.